odoo/.pylintrc

```diff
@@ -4,13 +4,22 @@ score=n
 
 [ODOOLINT]
 readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
+manifest_required_authors=Odoo Community Association (OCA)
 manifest_required_keys=license
 manifest_deprecated_keys=description,active
 license_allowed=AGPL-3,GPL-2,GPL-2 or any later version,GPL-3,GPL-3 or any later version,LGPL-3
+valid_odoo_versions=15.0
```

Comment:
We can't copy the file from the OCA 1 by 1. Internal manifests don't need to include OCA as author and the valid odoo version isn't always 15.0 because the template is used for all kind of Odoo versions.

Reply:
OCA as author check is disabled, AFAICT.
The valid Odoo versions can be disabled.